arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmio/MmioOperation.java

```diff
@@ -15,12 +15,24 @@
 
 package net.consensys.linea.zktracer.module.mmio;
 
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+import net.consensys.linea.zktracer.container.ModuleOperation;
 import net.consensys.linea.zktracer.module.hub.State;
 import net.consensys.linea.zktracer.runtime.microdata.MicroData;
 
-record MmioOperation(
-    MicroData microData,
-    MmioDataProcessor mmioDataProcessor,
-    State.TxState.Stamps moduleStamps,
-    int microStamp,
-    boolean isInitCode) {}
+@RequiredArgsConstructor
+@Accessors(fluent = true)
+class MmioOperation extends ModuleOperation {
```

Comment:
If all attributes are `@Getter`-annotated, just annotate the class itself with `@Getter`